recipes/bliss/all/patches/0001-patch-sources.patch

```diff
@@ -0,0 +1,177 @@
+--- bliss.cc
++++ bliss.cc
+@@ -1,3 +1,8 @@
++#ifdef _MSC_VER
```

Comment:
I guess including `<iso646.h>` is better way to get support for alternative tokens

Reply:
Thanks for the tip!
https://docs.microsoft.com/en-us/cpp/c-runtime-library/iso646-operators?view=vs-2019